### PR TITLE
Feature/prime sdk based service hooks

### PR DESCRIPTION
**Pull request by poocart**
### Added Changes
- Updated `example` React code to latest changes
- Updated `@etherspot/prime-sdk` to version `1.3.4`

### Breaking Changes
- Removed `etherspot` dependency
- Removed `@etherspot/react-etherspot` dependency
- Updated `provider` prop as required for `<EtherspotTransactionKit />` component
- Method `getSdkForChainId` on `useEtherspot` hook replaced with `getSdk` that returns `Prime SDK` instance as `Promise`
- Method `connect` on `useEtherspot` is no longer available and not required to start SDK session
- Values `accountAddress` and `providerWalletAddress` are no longer on `useEtherspot` hook, please use `useWalletAddress` hook
- Value `sdk` on `useEtherspot` hook is no longer present, please use `getSdk` on `useEtherspot` hook

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -22,11 +22,9 @@
   "homepage": "https://github.com/etherspot/transaction-kit#readme",
   "dependencies": {
     "@etherspot/eip1271-verification-util": "^0.1.2",
-    "@etherspot/react-etherspot": "^1.2.0",
+    "@etherspot/prime-sdk": "^1.3.4",
     "buffer": "^6.0.3",
     "ethers": "^5.6.9",
-    "etherspot": "^1.41.2",
```
[the remark this answers is not in the document]
Comment (poocart) [reply package.json:28]: RIP

File: src/services/EtherspotSessionStorage.ts [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  }
+
+  getSession = (walletAddress: string) => {
+    let result = null;
+
+    try {
+      const raw = window.localStorage.getItem(`session-prime-${walletAddress}`);
+      result = raw ? JSON.parse(raw) : null
+    } catch (err) {
+      //
```
Comment (IAmKio) [inline src/services/EtherspotSessionStorage.ts:21]: We should add a warning or something here should the device or security not allow local storage, feels like a showstopper?

**Review by IAmKio — CHANGES_REQUESTED** [review]
Just one tiny thing - other than that - looks and feels better!

**Review by IAmKio — DISMISSED** [review]

File: src/services/EtherspotSessionStorage.ts (hunk 2 above)
Comment (poocart) [reply src/services/EtherspotSessionStorage.ts]: session storage removed completely per SDK team feedback

**Pull request merged**